README.md

```diff
@@ -82,7 +82,7 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/exerci
 
 See CONTRIBUTING.md for details.
 
-This is an extraction from https://github.com/exercism/x-api.
+This is extraction from https://github.com/exercism/x-api.
```

Comment:
I didn't mean to do this here. I rewrote x-api from scratch then extracted it to this. Let me remove this.